src/Parlot/Fluent/Parsers.cs

```diff
@@ -117,10 +117,20 @@ public class LiteralBuilder
         public Parser<long> Integer() => new IntegerLiteral();
 
         /// <summary>
-        /// Builds a parser that matches a floating point number.
+        /// Builds a parser that matches a floating point number represented as a <lang cref="decimal"/> value.
         /// </summary>
         public Parser<decimal> Decimal() => new DecimalLiteral();
 
+        /// <summary>
+        /// Builds a parser that matches a floating point number represented as a <lang cref="float"/> value.
+        /// </summary>
+        public Parser<float> Float() => new FloatLiteral();
```

Comment:
I don't know how often these get called, but could also return `FloatLiteral.Default` which is internal readonly static field *if these don't have state*

Reply:
A static instance would work, but it's only created a few times in a parser. I am not even sure you would call the same "Float" in multiple places, for instance you'd have a single higher level parser which would use it, like a `PrimaryExpression` made of integers, floats, strings ...

So I don't think it's worth it. I'll check others though.